Point-of-No-Return/Object.h

```diff
@@ -238,7 +280,19 @@ class Object
 	Object(const Object&&);
 	Object& operator =(const Object&&);
 
+	Vec2 GetPos()
+	{
+		return vec;
```

Comment:
```suggestion
		return position;
```
何が良いのかはわかりませんが、少なくとも `vec` よりはマシでは？

Reply:
8cac1f9a でrenameしました。不適切であれば戻すなり変えるなりして下さい。